example/metadata/lib/src/client.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:grpc/grpc.dart';
+
+import 'generated/metadata.pbgrpc.dart';
+
+class Client {
+  ClientChannel channel;
+  MetadataClient stub;
+
+  Client() {
+    channel = new ClientChannel('127.0.0.1', port: 8080);
```

Comment:
It would feel a bit more obvious to initialize these in `main` where `channel` is also closed.